### PR TITLE
TASK: Add release notes for Flow 3.1

### PR DESCRIPTION
**Pull request by kdambekalns**
NEOS-1713 #comment Release notes for Flow 3.1

### CODE REVIEW

**Comment by aertmann** [comment]
:+1: by reading

File: TYPO3.Flow/Documentation/TheDefinitiveGuide/PartV/ReleaseNotes/310.rst [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+What has changed
+----------------
+
+Flow 3.1 comes with numerous fixes and improvements. Here's a list of changes that might need special attention when
+upgrading.
+
+In general make sure to run the commands::
+
+ ./flow flow:cache:flush --force
+ ./flow core:migrate
```
Comment (dimaip) [inline TYPO3.Flow/Documentation/TheDefinitiveGuide/PartV/ReleaseNotes/310.rst]: Don't think there are any core migrations

**Comment by robertlemke** [comment]
:+1:

**Comment by robertlemke** [comment]
Added a hint to the core migration section. Can be merged now, I think.

File: TYPO3.Flow/Documentation/TheDefinitiveGuide/PartV/ReleaseNotes/310.rst (hunk 1 above)
Comment (kdambekalns) [inline TYPO3.Flow/Documentation/TheDefinitiveGuide/PartV/ReleaseNotes/310.rst]: Ok, I'll remove it here, it's explained later again.

**Pull request merged**